src/playwrightTestServer.ts

```diff
@@ -80,7 +80,7 @@ export class PlaywrightTestServer {
       resolvePath,
     });
     for (const message of report)
-      teleReceiver.dispatch(message);
+      await teleReceiver.dispatch(message);
```

Comment:
Not sure about this and the following 3 places. This means that when we receive events we'll now await onEnd and onExit. Do you see any disadvantage in it or would you be careful and `void` it as well?

Reply:
Upstream we void instead of await, so I guess we should do the same here.